src/mixxx.cpp

```diff
@@ -1211,6 +1232,15 @@ void MixxxMainWindow::connectMenuBar() {
                 m_pLibrary,
                 &Library::slotCreatePlaylist);
     }
+
+#ifdef __ENGINEPRIME__
+    if (m_pLibraryExporter) {
+        connect(m_pMenuBar,
+                &WMainMenuBar::exportLibrary,
+                m_pLibraryExporter.get(),
+                &mixxx::LibraryExporter::slotRequestExport);
+    }
```

Comment:
Can you make this a `VERIFY_OR_DEBUG_ASSERT`?

Reply:
This now asserts - fixed.